### ISSUE TITLE
Ctrl-V: Double Paste (Content pasted twice into notable) 

### ISSUE THREAD

**Issue by smashingchrische** [notable/notable#925]
- **OS Version**: Windows 10 (1903)
- **Notable Version**: 1.8.1

### Current behavior
When text is pasted into the notable application either using Ctrl-V Shortcut
or Right-Click Paste Option from the menu. The content gets pasted twice.
This behavior is only related to notable. A paste into another application like the
standard notepad executable has any issue.
This behavior occurred only after the use of notable directly after installation.
A restart of the system fixed this issue.  

### Expected behavior
Single paste of content.

clipboard content: test
Ctrl-V in notable: testtest
expected: test

### Remarks
The issue occurred on both PCs directly after installation.
A simple restart of the system fixed the issue.
I am not sure whether it is a bug or you require a system restart.

And thanks for the application I really like it.

**Comment by stickyPiston** [comment]
I experience the same issue. Using the latest notable version on macOS Catalina.

**Comment by fabiospampinato** [comment]
Duplicate of #779. For now restart the app to fix this temporarily.
